tools/wpt/tests/test_revlist.py

```diff
@@ -56,14 +58,7 @@ def test_get_epoch_revisions(mocked_check_output):
 merge_pr_A A 388800 _mon_
 '''
     tagged_revisions = revlist.get_epoch_revisions(epoch, until, 8)
-    assert tagged_revisions.next() == 'N'
```

Comment:
The `next()` built-in was ported back to python2, wasn't it?

```
Python 2.7.17rc1 (default, Oct 10 2019, 10:26:01) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> iterator = iter([1, 2, 3])
>>> next(iterator)
1
>>> next(iterator)
2
>>> next(iterator)
3
>>> next(iterator)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
StopIteration
```

To minimize the code-diff here, could we use that instead?

Reply:
Sure, but isn't the ['a', 'b', 'c'] format a bit more readable? No strong opinion there though

Reply:
I agree with @svillar .

Reply:
Sgtm